### DISCUSSION TITLE
How to increase buffer completion length limit?

### DISCUSSION THREAD

**Discussion by kik4444** [saghen/blink.cmp#474] [Q&A]
Hello, I just trying blink for the first time from LazyVim. This is the configuration I'm using with LazyVim at the time of writing: https://github.com/LazyVim/LazyVim/blob/v13.8.0/lua/lazyvim/plugins/extras/coding/blink.lua

However, I noticed that there seems to be a limit on the completion length of buffer items. For example, if I open a new empty file, write a long word on the first line, then attempt to autocomplete it on the second line, I get this:
```
Thisisalongwordthatwearetryingtoreach
Thisisalongwordthatwearetryingtor
```
The first line is manually written. The second was autocompleted by blink. It got cut off at the 33rd character.

This happens in other files too that have multiple completion sources. But it only happens with the `buffer` completion. If I have a long function name, then the LSP completion is full and not cut off.

I didn't notice any settings for buffer length completions, but I imagine there must be something which is why I'm asking here instead of making an issue.

Any ideas how to raise this length limit, assuming it's configurable?

**Comment by saghen** [comment]
It's hard coded in the rust but it should be easy to make it configurable. Do you mind making an issue for this?
